dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -624,3 +624,79 @@ CWWKD1063.unsupported.resource.explanation=The Jakarta Persistence provider does
  not support the unwrap operation returning the specified resource type.
 CWWKD1063.unsupported.resource.useraction=Remove the unsupported resource accessor \
  method from the repository interface.
+
+CWWKD1064.datastore.error=CWWKD1064E: The {0} method of the {1} repository \
+ interface is unable to obtain the {2} data store for the repository due to \
+ the following error: {3}.
+CWWKD1064.datastore.error.explanation=An error occurred when attempting to obtain \
+ the data store.
+CWWKD1064.datastore.error.useraction=Ensure that the specified data store and all \
+ other resources it depends on exist and are correctly configured.
+
+CWWKD1065.ddlgen.timeout=CWWKD1065E: DDL generation for the {0} repository \
+ interface failed because the {1} data store did not complete DDL generation \
+ within {2} seconds for the following entities: {3}.
+CWWKD1065.ddlgen.timeout.explanation=The DDL generation attempt timed out \
+ and did not complete.
+CWWKD1065.ddlgen.timeout.useraction=Reattempt DDL generation \
+ when the system and database are not under load.
+
+CWWKD1066.ddlgen.failed=CWWKD1066E: DDL generation for the {0} repository \
+ interface that uses the {1} data store encountered an error when attempting \
+ DDL generation for the following entities: {2}. The error is: {3}.
+CWWKD1066.ddlgen.failed.explanation=The DDL generation attempt failed \
+ and did not complete.
+CWWKD1066.ddlgen.failed.useraction=Investigate the error and make corrections \
+ to the repository interface, entities, data store configuration, or database.
+
+CWWKD1067.ddlgen.emf.timeout=CWWKD1067E: DDL generation for the {0} repository \
+ interface failed because the {1} data store was not available or was unable \
+ to make the following entities available within {2} seconds: {3}.
+CWWKD1067.ddlgen.emf.timeout.explanation=Acquisition of the resources needed \
+ for DDL generation timed out and did not complete.
+CWWKD1067.ddlgen.emf.timeout.useraction=Reattempt DDL generation \
+ when the system and database are not under load.
+
+CWWKD1068.entity.name.conflict=CWWKD1068E: The {0} repository interface has \
+ multiple entities named {1} which are provided by different classes: {2}. \
```

Comment:
Replace "which" with "that" or add a comma before "which"

Reply:
Fixed to use "that" in https://github.com/OpenLiberty/open-liberty/pull/29696/commits/b6267bdfd34502bb71c25d6d609cfc44284df899